components/ws-manager/pkg/manager/create.go

```diff
@@ -592,6 +592,21 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 			gitpodGUID := int64(133332)
 			pod.Spec.SecurityContext.FSGroup = &gitpodGUID
 
+			// add init container to chown workspace subpath, so that it is owned by gitpod user (there is no k8s native way of doing this as of right now)
+			pod.Spec.InitContainers = append(pod.Spec.InitContainers, corev1.Container{
```

Comment:
Sorry if this question is too beginner, but why do we need to use InitContainer instead of a normal container?

Reply:
@utam0k we only need to do chown at container startup, and it ensures it runs before our workspace container. if we do as normal container, then both containers would start at the same time.